server/src/controllers/noncommercial.es6

```diff
@@ -413,12 +414,13 @@ noncommercial.create = (req, res) => {
   };
   translateFromClientToDatabase(req.body, model);
   NoncommercialApplication.create(model)
-    .then(noncommApp => {
-      util.logControllerAction(req, 'noncommerical.create', noncommApp);
-      email.sendEmail('noncommercialApplicationSubmittedAdminConfirmation', noncommApp);
-      email.sendEmail('noncommercialApplicationSubmittedConfirmation', noncommApp);
-      req.body['applicationId'] = noncommApp.applicationId;
-      req.body['appControlNumber'] = noncommApp.appControlNumber;
+    .then(app => {
+      util.logControllerAction(req, 'noncommerical.create', app);
+      app.forestName = forestInfoService.specialUseForestName(app.region+app.forest);
```

Comment:
Minor style question - should there be spaces around the `+`?  (For consistency, since there are spaces elsewhere.)